src/utils.py

```diff
@@ -166,3 +168,33 @@ def printyellow(text):
     reset = "\033[0m"
     logger.debug("Printing text in yellow: %s", text)
     print(f"{yellow}{text}{reset}")
+
+
+def capture_error(driver, error_message):
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+    error_dir = os.path.join("error_logs", timestamp)
+    os.makedirs(error_dir, exist_ok=True)
+
+    # Save the error message to a log file
+    log_file_path = os.path.join(error_dir, "error.log")
+    with open(log_file_path, "w") as log_file:
+        log_file.write(error_message)
+
+    # Capture a screenshot
+    screenshot_path = os.path.join(error_dir, "screenshot.png")
+    driver.save_screenshot(screenshot_path)
+
+    # Record a video of the browser screen
```

Comment:
move line 187 to 196 to separate method - call it as record browser screen, you  can have time as param (up to you)